src/engine/renderer/tr_shader.cpp

```diff
@@ -3706,236 +3706,187 @@ CollapseMultitexture
 // *INDENT-OFF*
 static void CollapseStages()
 {
-	int           i, j;
-	int           numStages = 0;
-
 	if ( !r_collapseStages->integer )
 	{
 		return;
 	}
 
-	shaderStage_t tmpStages[ MAX_SHADER_STAGES ]{};
```

Comment:
hwo come 
```
shaderStage_t tmpStages[ MAX_SHADER_STAGES ] = {};
```

doesn't work?

Reply:
I tried that but still got the same looking backtrace.